src/main/resources/reference.conf

```diff
@@ -998,14 +998,29 @@ resourceTypes = {
       "alter_policies" = {
         description = "Can alter policies"
       }
+      "add_child" = {
+        description = "add a child resource"
+      }
+      "remove_child" = {
```

Comment:
Landing zones can be deleted by the owner. Not sure if there is a need for `remove_child` at the parent level though, since billing profile owners inherit owner on the landing zone. I can remove this..

Reply:
deleting a LZ is different than removing its parent